pandas/io/formats/format.py

```diff
@@ -84,12 +84,23 @@
                              "match-parent", "initial", "unset")
 
 justify_docstring = """
-    justify : {'left', 'right', 'center', 'justify',
-               'justify-all', 'start', 'end', 'inherit',
-               'match-parent', 'initial', 'unset'}, default None
+    justify : str, default None
         How to justify the column labels. If None uses the option from
         the print configuration (controlled by set_option), 'right' out
-        of the box."""
+        of the box. Valid values are
+
+        * left
```

Comment:
@jorisvandenbossche Thoughts on this one? The multiline `{ options... }` format was giving warnings (block quotes I think?). I couldn't figure out the correct values in the formatting, so I just made a list.

Reply:
See https://github.com/numpy/numpydoc/issues/87. In the end I think the conclusion is that using `\` is the best way to deal with multi-line type specifications. 

On the other hand, if the list becomes too long of all options, I think it is good to not list them all in the type specification (but keep it there to simply 'string') but give more explanation in the description.